jmeter-java-dsl-dashboard/src/test/java/us/abstracta/jmeter/javadsl/dashboard/DashboardVisualizerTest.java

```diff
@@ -34,5 +45,21 @@ public void shouldDisplayGraphsAndSummaryWhenRunTestPlanWithDashboard(TestInfo t
         testInfo
     );
   }
+    @Test
+    public void testShowTestElementGui() {
```

Comment:
Even though showTestElementGui is public, it is not intended for general user usage, it was just made public for other part of JMeter DSL to be able to use it. Additionally, I think this logic is in part tested in previous test. So in general I think this test could be removed and adds little value to the test set (considering that it adds code to maintain and time for test suite execution).